Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -121,10 +155,30 @@ open class ChatChannelListVC: _ViewController,
                 height: 64
             )
         }
+    
+        channelListErrorView.buttonAction = { [weak self] in
+            self?.controller.synchronize()
+        }
+    }
+
+    /// Shows / Hides the error view. This error view is displayed at the bottom by default
+    /// - Parameter show: A boolean which denotes whether to show or hide the error view
+    ///
+    /// The error view is shown when some error occurs while loading channels
+    open func showErrorView(_ show: Bool) {
+        channelListErrorView.isHidden = !show
+        UIView.animate(withDuration: 0.8) {
```

Comment:
magic number should be moved to private enum or class constant
>0.8

Reply:
It is not magic by definition because it's used in a context that literally explains that this number is animation duration and it's used in a single place